app/metrics.yaml

```diff
@@ -3131,3 +3131,61 @@ perf.awesomebar:
       - fenix-core@mozilla.com
       - gkruglov@mozilla.com
     expires: "2020-10-01"
+
+progressive_web_app:
+  homescreen_tap:
+    type: event
+    description: |
+      A user taps on PWA homescreen icon
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/10261
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11859
+    notification_emails:
+      - fenix-core@mozilla.com
+      - erichards@mozilla.com
+    expires: "2020-10-01"
+  install_tap:
+    type: event
+    description: |
+      A user installs a PWA. Could be a shortcut or added to homescreen.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/10261
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11859
+    notification_emails:
+      - fenix-core@mozilla.com
+      - erichards@mozilla.com
+    expires: "2020-10-01"
+  foreground:
```

Comment:
Should these keys be changed to something like `intent_receiver_foreground` to make it clear it doesn't refer to the actual PWA foreground time?